test/test_view_drawing.py

```diff
@@ -0,0 +1,95 @@
+import os
```

Comment:
No file doco, however considering none of the other tests do it should be sweet

Reply:
noted